pom.xml

```diff
@@ -28,7 +28,7 @@
     <version>1.2.1-SNAPSHOT</version>
     <name>EASY Deposit API</name>
     <url>https://github.com/DANS-KNAW/easy-deposit-api</url>
-    <description>JSON-based deposit service.</description>
+    <description>This service offers a JSON-based deposit API</description>
```

Comment:
shouldn't this be the same as the `description` in `CommandLineOptions`? there should be a test for this, right? Or at least it is the case in other modules; not sure about this 😉

Reply:
The test whether the description of the `CommandLineOptions` is found in the pom. The pom can have extra's.